sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcUtils.scala

```diff
@@ -372,12 +385,14 @@ object OrcUtils extends Logging {
       canPruneCols: Boolean,
       dataSchema: StructType,
       resultSchema: StructType,
+      orcCatalystSchema: StructType,
       partitionSchema: StructType,
       conf: Configuration): String = {
     val resultSchemaString = if (canPruneCols) {
-      OrcUtils.orcTypeDescriptionString(resultSchema)
```

Comment:
so the code change can be
```
val actualResultSchema = replaceNTZType(resultSchema)
... the original code
```

Reply:
When reading ntz as ltz, `resultSchema` not contains timestamp NTZ.